src/tokenizer.c

```diff
@@ -82,14 +82,14 @@ typedef struct GumboInternalTagState {
   // of the buffer.
   const char* _original_text;
 
-  // The current tag enum, computed once the tag name state has finished so that
+  // The ui tag enum, computed once the tag name state has finished so that
```

Comment:
Yes it's a simple find&replace. Im using gumbo as a submodule in my own project, and modified these files by accident when i was doing a refactoring of my code. And a later commit had restored those links to there original value.